docs/working-with-projects/working-with-images/preheat-images.md

```diff
@@ -2,12 +2,11 @@
 title: Preheat Images
 weight: 30
 ---
-{{< note >}}
+
 Before you can preheat images, a system administrator must first configure a P2P provider instance. See more about [configuring P2P preheating in Harbor](../../administration/p2p-preheat/manage-preheat-providers.md).
```

Comment:
This link is broken, should go to administration/p2p-preheat/manage-preheat-providers/

Reply:
updated